app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -1214,6 +1214,42 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPassword_withAdmin_inputPin_configChange_pinIsPersisted() {
```

Comment:
Nit: Slight naming suggestion to align with other test cases.
```suggestion
  fun testPinPassword_withAdmin_inputPin_configChange_inputPinIsPersisted() {
```